### PR TITLE
Added Monica #288

### PR DESCRIPTION
**Pull request by kgnfth**
### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/monica.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+captainVersion: 4
+services:
+    # Monica
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-mariadb
+        image: monica:apache
```
Comment (githubsaturn) [inline public/v4/apps/monica.yml:7]: I am assuming they don't have versioned tags, right?

If that's the case, I suggest we use sha hash to ensure that the one-click app won't break if they change the image and change the requirement in the future.

File: public/v4/apps/monica.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        environment:
+            APP_KEY: $$cap_gen_random_hex(32)
+            DB_HOST: srv-captain--$$cap_appname-mariadb
+            DB_DATABASE: $$cap_mariadb-db
+            DB_USERNAME: $$cap_mariadb-user
+            DB_PASSWORD: $$cap_mariadb-pass
+        volumes:
+            - $$cap_appname-storage:/var/www/html/storage
+        caproverExtra:
+            containerHttpPort: '80'
```
Comment (githubsaturn) [inline public/v4/apps/monica.yml:18]: minor: port 80 is the default and not needed.

**Comment by githubsaturn** [comment]
Thanks!

**Comment by aniketpanjwani** [comment]
I'm on CapRover 1.8.2, and this one-click app isn't working for me. After deployment, I'm getting an NGINX 502 Error page with a link to [these docs](https://caprover.com/docs/troubleshooting.html#successful-deploy-but-502-bad-gateway-error). I tried deploying another one-click app, CodiMD, to make sure the problem isn't with all one-click apps, and that app works fine. The only thing I modified in the one-click app configs for Monica was the name of the app, which was left blank, so I called it "monica".

Don't have time for a full trouble-shooting now, but I thought I'd make a quick comment in case anyone else is experiencing similar issues.

**Comment by aniketpanjwani** [comment]
I think I figured out the issue - I was leaving the MariaDB password blank, but that's required. Got it working now!

**Comment by kgnfth** [comment]
Works fine here http://monica.app.gocloud.fun

**Comment by kgnfth** [comment]
Works fine here https://monica.app.gocloud.fun

to fix your styles you have to add `APP_ENV=production` in your app env config

See https://hub.docker.com/_/monica?tab=description and scroll down all the way to `Make Monica available from the internet`

**Comment by aniketpanjwani** [comment]
@kgnfth Thanks that fixed it for me!

**Comment by githubsaturn** [comment]
@kgnfth  - Thanks for pointing out `APP_ENV=production` 

Is there any downside to including this in the default config?

**Comment by kgnfth** [comment]
@githubsaturn No, but i think i missed this option when i first created this template
i will send a pull now

**Pull request merged**